server/export.js

```diff
@@ -49,6 +49,15 @@ export default async function (dir, options, configuration) {
     )
   }
 
+  // Copy .next/static directory
+  if (existsSync(join(dir, '.next', 'static'))) {
```

Comment:
I believe you want to use `distDir` from `next.config.js` instead of hardcoding `.next`

i.e. `join(nextDir, 'static')` in this scope

Reply:
Good point.

Reply:
Fixed 🙏